cve_bin_tool/cli.py

```diff
@@ -139,6 +141,18 @@ def main(argv=None):
     if args.quiet:
         LOGGER.setLevel(logging.CRITICAL)
 
+    # Check for the latest version
+    name = "cve-bin-tool"
+    url = f"https://pypi.org/pypi/{name}/json"
```

Comment:
Let's make sure we have a command line flag that allows us to opt out of checking for the latest version